libs/core/src/components/pds-popover/docs/pds-popover.mdx

```diff
@@ -0,0 +1,101 @@
+import { DocArgsTable, DocCanvas } from '@pine-ds/doc-components';
+import { components } from '../../../../dist/docs.json';
+
+# Popover
+
+Popovers display contextual information when users click or fires a keyboard event on an element. They typically contain more information than tooltips and can include interactive elements.
+
+## Guidelines
+### Usage
+- Use popovers for complex information that might overwhelm the main content.
+- Consider using a tooltip for simpler explanations or brief labels.
+
+#### When not to use
+- Don't use popovers for critical information that must be readily available.
+
+## Properties
+
+<DocArgsTable componentName="pds-popover" docSource={components} />
+
+## Variants
+### Default
+The popover's appearance and behavior are primarily controlled by two slots:
+1. **`trigger`:** This slot is used to define the element that triggers the popover. It can be any HTML element, such as a button or link.
```

Comment:
```suggestion
1. **`trigger`:** This slot specifies the interactive element that activates the popover. It can be any HTML element, such as a button, link, or icon, that users interact with to reveal the popover. Ensure the trigger clearly indicates its purpose and is accessible to all users.
```

Recommended a change to avoid using the word in the definition :)